iOS/Multiplatform Redux Sample/Multiplatform Redux Sample/Utils/PanGestureView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  PanableGestureView.swift
+//  Multiplatform Redux Sample
+//
+//  Created by Samuel Bichsel on 09.03.21.
+//
+
+import UIKit
+
+protocol PanGestureViewDelegate: class {
+    func hide(view: PanGestureView)
+}
+
+class PanGestureView: UIView {
+    static let minimalCloseVelocity: CGFloat = 1500
+    static let minimalCloseHeightFactor: CGFloat = 0.3
+    static let animationDuration: TimeInterval = 0.3
```

Comment:
Is there a reason why these aren't declared `private`? If not consider declaring them at file level.